### PR TITLE
Template updates and tweaks

### PR DESCRIPTION
**Pull request by jcape**
- [x] Add `settings.yml`
- [x] Add `javascript` label
- [x] Add `dependencies` label
- [x] Run auto-assign on draft PRs
- [x] Fix single-crate repo package metadata
- [x] Use taki-e/install-action

### Motivation

This is a collection of minor fixes, along with adding settings.yml.

### CODE REVIEW

**Comment by jcape** [comment]
Current dependencies on/for this PR:
* main
  * **PR #45** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/cookiecutters/45" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/cookiecutters/45?utm_source=stack-comment).

File: rust/repo/examples/default/output/repository/.github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    color: '#ff9100'
+    description: Extra-Large PRs
+  - name: size/XXL
+    color: '#f24d11'
+    description: Double-wide PRs
+  - name: size/OHLAWDHECOMIN
+    color: '#ed1717'
+    description: PRs that should get broken down
+
+collaborator:
```
Comment (nick-mobilecoin) [inline rust/repo/examples/default/output/repository/.github/settings.yml:65]: :wrench: The one in `.github/settings.yml` should be updated as well
```suggestion
collaborators:
```

File: rust/repo/{{ cookiecutter.repo_name }}/.github/settings.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    color: '#ff9100'
+    description: Extra-Large PRs
+  - name: size/XXL
+    color: '#f24d11'
+    description: Double-wide PRs
+  - name: size/OHLAWDHECOMIN
+    color: '#ed1717'
+    description: PRs that should get broken down
+
+collaborator:
```
Comment (nick-mobilecoin) [inline rust/repo/{{ cookiecutter.repo_name }}/.github/settings.yml:65]: :wrench: 
```suggestion
collaborators:
```

**Review by nick-mobilecoin — CHANGES_REQUESTED** [review]
Mainly need agreement on `collaborator` vs `collaborators` property. It maybe that both are accepted, but the example at https://github.com/apps/settings uses the plural.

**Pull request merged**